### PR TITLE
[ExecutableInstallation] Update links

### PR DESCRIPTION
**Pull request by Herklos**

### CODE REVIEW

File: _i18n/en.yml [hunk 1]
```diff
@@ -392,10 +392,11 @@ installations:
...
     release_example: Downloadable executables for the 0.4.0-beta8 version
     step1: |
-      1. Go to <a id="executable-installation-go-to-releases"
-      href="https://github.com/Drakkar-Software/OctoBot/releases/latest" target="_blank" rel="noopener">
-      OctoBot's official releases page</a> and download the latest version for your system: Windows, Linux or MacOS.
-      You can even use OctoBot on <a href="https://www.raspberrypi.org/"target="_blank" rel="noopener"> Raspberry pie</a>.
+      1. Download the latest version for your system:</br>
+      - <a id="executable-installation-download-windows" href="https://release.octobot.online/stable/OctoBot_windows_x64.exe" target="_blank" rel="noopener">Windows</a></br>
+      - <a id="executable-installation-download-windows" href="https://release.octobot.online/stable/OctoBot_linux_x64" target="_blank" rel="noopener">Linux</a></br>
+      - <a id="executable-installation-download-windows" href="https://release.octobot.online/stable/OctoBot_macos_x64" target="_blank" rel="noopener">MacOS</a></br>
```
Comment (GuillaumeDSM) [inline _i18n/en.yml:398]: executable-installation-download-linux
executable-installation-download-macos

(same in fr)

File: _i18n/en.yml [hunk 2]
```diff
@@ -392,10 +392,11 @@ installations:
...
+      You can even use OctoBot on <a href="https://release.octobot.online/stable/OctoBot_linux_arm64"target="_blank" rel="noopener">Raspberry pie</a> with an ARM64 OS.
```
Comment (GuillaumeDSM) [inline _i18n/en.yml:399]: the id attr is missing

**Review by GuillaumeDSM — APPROVED** [review]
It would be even better with OS images, maybe we should add it after the marketing tests ?

**Comment by Herklos** [comment]
> It would be even better with OS images, maybe we should add it after the marketing tests ?

You mean adding those links after the marketing tests or OS images ?

**Review by GuillaumeDSM — APPROVED** [review]

**Comment by GuillaumeDSM** [comment]
> > It would be even better with OS images, maybe we should add it after the marketing tests ?
> 
> You mean adding those links after the marketing tests or OS images ?

I mean having the links displayed as images of the associated OS.

**Pull request merged**